provenance-api/resources/mongo_js.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
This script was previously generated inside the docker file via echo statements.